content/spin/v2/javascript-components.md

```diff
@@ -385,12 +326,45 @@ For more information about using relational databases from TypeScript/JavaScript
 
 For more information about using Serverless AI from JS/TS, see the [Serverless AI](serverless-ai-api-guide) API guide.
 
+## `Node.js` Compatibility
+
+The SDK does not support the full specification of `Node.js`. A limited set of APIs can be polyfilled using the [`@fermyon/wasi-ext`](https://github.com/fermyon/js-wasi-ext) library which provided a webpack plugin. It can be used by installing the  library first using:
```

Comment:
"library which provided a webpack plugin"? Why the past tense?

Reply:
typo - Should have been provides. will fix.